client/python/gradio_client/client.py

```diff
@@ -93,6 +109,67 @@ def fn(future):
 
         return job
 
+    def usage(self, all_endpoints=True, print_usage=True) -> str | None:
```

Comment:
What if we call the method `view_api`? I like that it matches what a user would see when opening the UI.